### DISCUSSION TITLE
Broken link?

### DISCUSSION THREAD

**Discussion by wileycoyote78** [wp-cli/wp-cli#5905] [General]
This URL ( https://github.com/wp-cli/wp-cli/wiki/Commands-Cookbook ) has a notice that it is deprecated in favor of this URL ( http://wp-cli.org/docs/commands-cookbook/ ), but the favored URL lands in a 404. Just thought someone would like to know.

**Comment by danielbachhuber** [comment]
Thanks for the heads up! I fixed the link.
